server/utils.go

```diff
@@ -7,9 +7,19 @@ import (
 // nextWeekdayDate calculates the date of the next given weekday
 // based on today's date.
 // If nextWeek is true, it will be based on the next calendar week.
-func nextWeekdayDate(day time.Weekday, nextWeek bool) time.Time {
+func nextWeekdayDate(meetingDays []time.Weekday, nextWeek bool) time.Time {
+	todayWeekday := time.Now().Weekday()
+
+	// Find which meeting weekday to calculate the date for
+	meetingDay := meetingDays[0]
```

Comment:
Should we handle the possibility that `len(meetingDays) == 0`?

Reply:
Added length check.